packages/cli-upload/src/commands/upload.js

```diff
@@ -10,7 +10,7 @@ import createImageResources from '../resources';
 import { schema } from '../config';
 import pkg from '../../package.json';
 
-const ALLOWED_FILE_TYPES = /\.(png|jpg|jpeg)$/i;
+const ALLOWED_FILE_TYPES = /^\.(png|jpg|jpeg)$/i;
```

Comment:
@wwilsman Should this also include uppercase for windows users where windows defaults the file type extension to upper case.

Reply:
@andressalinero no need, as the `i` at the end of the regular expression means it's case insensitive.

Reply:
That's very elegant!